src/components/Timepicker/Timepicker.const.tsx

```diff
@@ -0,0 +1,25 @@
+import { endOfDay } from 'date-fns/esm';
+import { nlBE } from 'date-fns/locale';
+import { ReactDatePickerProps } from 'react-datepicker';
+
+const now = new Date();
+
+const base: Partial<ReactDatePickerProps> = {
+	dateFormat: 'HH:mm',
+	locale: nlBE,
```

Comment:
Idem comment bij #24 , dit zijn zaken die in de client gaan gedefinieerd worden dan hebben we hier geen extra deps op `date-fns` nodig.